dags/common/task_group_with_timeout/task_group_with_timeout.py

```
@@ -204,3 +213,37 @@ def _determine_task_timeout(task: BaseOperator) -> float:
+      task_retry_delay,
+      group_deadline,
+      group_name,
+      seconds=1,
+      error_message="Timeout",
+  ):
+    super().__init__(seconds=seconds, error_message=error_message)
+    self.group_deadline = group_deadline
+    self.task_retry_delay = task_retry_delay
+    self.group_name = group_name
+
+  def handle_timeout(self, *args):
```

Ruff (0.16.2)

[warning] Missing type annotation for `*args`

(ANN002)